src/middleware/getPhotos.js

```diff
@@ -10,17 +10,24 @@
   See the License for the specific language governing permissions and
```

Comment:
Adding error clarity here as well as injected assurance that table exists with ability to create it if not.